addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -1,20 +1,26 @@
 odoo.define('website.s_facebook_page', function (require) {
 'use strict';
 
-var publicWidget = require('web.public.widget');
-var utils = require('web.utils');
+const publicWidget = require('web.public.widget');
+const utils = require('web.utils');
+const { debounce } = require("@web/core/utils/timing");
 
 const FacebookPageWidget = publicWidget.Widget.extend({
     selector: '.o_facebook_page',
     disabledInEditableMode: false,
 
+    /**
+     * @constructor
+     */
+    init() {
+        this._super(...arguments);
+        this.previousWidth = 0;
+    },
     /**
      * @override
      */
-    start: function () {
-        var def = this._super.apply(this, arguments);
-
-        this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerUnactive();
+    start() {
+        const def = this._super(...arguments);
```

Comment:
Don't change those lines if there is no diff if you don't convert them to ES6

Reply:
@bvr-odoo That's why we don't touch any other code until it is really a part of our diff.
Here you suggested to change ```var``` to ```const```
https://github.com/odoo/odoo/pull/163739/files#r1639442017
So we thought let's change other things to ES6 as well.